### PR TITLE
Add announcement for 2.1

### PR DESCRIPTION
**Pull request by parisk**

### CODE REVIEW

File: _posts/2016-11-10-xtermjs-2.1.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title:  Announcing xterm.js 2.1
+author: parisk
+author_twitter: pariskasid
+categories: xtermjs update
+excerpt: We are excited to announce xterm.js 2.1 with all-new scrolling API, Terminado support and more!
+---
+
+We are announcing **<a href="https://github.com/sourcelair/xterm.js" target="_blank">xterm.js 2.1</a>**! Below you can find some the most important updates of this release.
```
Comment (Tyriar) [inline _posts/2016-11-10-xtermjs-2.1.md]: We are announcing ->
Today we are announcing

Below you can find some the most ->
Below you can find some of the most

File: _posts/2016-11-10-xtermjs-2.1.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## 🆕 All-new scrolling API
```
Comment (Tyriar) [inline _posts/2016-11-10-xtermjs-2.1.md]: Will that new icon be rendered on older browsers/OS'?

File: _posts/2016-11-10-xtermjs-2.1.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+In addition to line scrolling, we are introducing a few higher level methods, that let you manipulate scrolling with more ease:
```
Comment (Tyriar) [inline _posts/2016-11-10-xtermjs-2.1.md]: that let you manipulate scrolling with more ease ->
that give you finer-grained control on how much you want to scroll.

File: _posts/2016-11-10-xtermjs-2.1.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+- [`scrollPages(n)`](/docs/api/Terminal/#scrollpagesn): Scrolls viewport down by `n` pages (or scrolls up if `n` is negative)
+- [`scrollToTop()`](/docs/api/Terminal/#scrolltotop): Scrolls the terminal to the top of the buffer
+- [`scrollToBottom()`](/docs/api/Terminal/#scrolltobottom): Scrolls the terminal to the bottom of the buffer.
+
+## 🆕 Terminado addon
+In this release we included a new [Terminado](http://terminado.readthedocs.io/en/latest/) addon. Terminado exposes back-end terminals via a WebSocket protocol. To learn how you can hook an xterm.js front-end to a Terminado back-end take a look at our [Connecting to Terminado](/docs/guides/terminado/) guide  or to the source code of the [`terminado`](https://github.com/sourcelair/xterm.js/blob/2.1.0/addons/terminado/terminado.js) addon.
```
Comment (Tyriar) [inline _posts/2016-11-10-xtermjs-2.1.md]: guide  or to the source code of the ->
guide or at the source code of the

File: _posts/2016-11-10-xtermjs-2.1.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## IME improvements
+We bundled multiple improvements to our IME support in this release, as it is one of xterm.js' most important features. In more details:
+
+- Hidden textarea is correctly positioned when IME is inactive ([#301](https://github.com/sourcelair/xterm.js/pull/301))
+- The composition view is always correctly sized and positioned ([#320](https://github.com/sourcelair/xterm.js/pull/320))
+- IME position is correct and stabilized when active ([#323](https://github.com/sourcelair/xterm.js/pull/323))
+
+## Right-click, copy and paste improvements
+Getting right-click and clipboard manipulation right across all major browser is a tough task. In this release we did great progress towards this, with the following updates:
```
Comment (Tyriar) [inline _posts/2016-11-10-xtermjs-2.1.md]: In this release we did great progress ->
In this release we made some great progress

File: _posts/2016-11-10-xtermjs-2.1.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+- <a href="https://github.com/onttoni"> <img class="avatar mr-2" src="https://github.com/onttoni.png?s=32" /> onttoni</a> for helping [fix auto-scrolling on output](https://github.com/sourcelair/xterm.js/pull/336/commits/5e68acfc761870930942e057b0c61fcde722bfb1)
+- <a href="https://github.com/IanLewis"> <img class="avatar mr-2" src="https://github.com/IanLewis.png?s=32" /> IanLewis</a> for [adding the `main` field in owr `bower.json` file](https://github.com/sourcelair/xterm.js/pull/337)
+- <a href="https://github.com/hiro-su"> <img class="avatar mr-2" src="https://github.com/hiro-su.png?s=32" /> hiro-su</a> for fixing [copy](https://github.com/sourcelair/xterm.js/pull/339), [paste](https://github.com/sourcelair/xterm.js/pull/334) and [right-click paste](https://github.com/sourcelair/xterm.js/pull/345) on Internet Explorer
+
+Xterm.js 2.1 is available at [npm](http://npmjs.org/package/xterm), [bower](https://bower.io) and [GitHub Releases](https://github.com/sourcelair/xterm.js/releases/tag/2.1.0) as usual, so go get it and create amazing terminal apps now!
```
Comment (Tyriar) [inline _posts/2016-11-10-xtermjs-2.1.md]: available at ->
available on

so go get it and create amazing ->
so go get it and start creating amazing

**Comment by parisk** [comment]
Fixed comments. Thanks @Tyriar!

Moving on with merging.

**Pull request merged**